diy-omd.ino

```diff
@@ -16,38 +20,54 @@ DallasTemperature temperatureSensors(&oneWire);
 elapsedMillis timeMillis;
 elapsedMillis secondaryTimeMillis;
 elapsedMillis tempTimeMillis;
+elapsedMillis testMillis;
```

Comment:
We're getting a lot of variables tracking time now, can we name it to specifically reference what it's for? Maybe "bluetoothTimeMillis"?